src/app/CommandSender.cpp

```diff
@@ -121,7 +121,7 @@ CHIP_ERROR CommandSender::ProcessCommandDataElement(CommandDataElement::Parser &
     chip::EndpointId endpointId;
     Protocols::SecureChannel::GeneralStatusCode generalCode = Protocols::SecureChannel::GeneralStatusCode::kSuccess;
     uint32_t protocolId                                     = 0;
-    uint16_t protocolCode                                   = 0;
+    uint32_t protocolCode                                   = 0;
```

Comment:
I think the encoding spec is incorrect here in specifying this as a 32-bit number - I believe this is a 16-bit number. Will check with Cam.

Reply:
Per offline chat, confirmed that it is 16bit. 
https://github.com/CHIP-Specifications/connectedhomeip-spec/issues/4053